app/controllers/steps/supporting_evidence_controller.rb

```diff
@@ -3,43 +3,41 @@
 module Steps
   class SupportingEvidenceController < Steps::BaseStepController
     skip_before_action :verify_authenticity_token
-    before_action :supporting_evidence
+    before_action :supporting_evidence, :file_uploader
```

Comment:
we don't need these here if we use the method to access it (over the `@` variable as it will just lazy load... :)